### DISCUSSION TITLE
Kaboom Headless

### DISCUSSION THREAD

**Discussion by DaInfLoop** [replit/kaboom#545] [Ideas]
Would be great if you wanted to use Kaboom in an actual node project without esbuild etc.

Would be used like:
```js
import kaboom from 'kaboom';

kaboom({
  // other settings...
  headless: true
})
```

and then you can access the canvas with `KaboomCtx#canvas`.

**Comment by slmjkdbtl** [comment]
Yeah headless is on my mind, will approach some point in the future (it'll be a pretty big refactor)
